firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Bytes.txt

```diff
@@ -0,0 +1,59 @@
+Scenario:The Util get_byte function correctly gets bytes from the given position
+   GIVEN("The util get_byte function")
+       WHEN("the get_byte function is called on 0x86 ")
+          THEN(""The value at the least significant byte position is "0x0a'")
+   GIVEN("The util get_byte function " )
+       WHEN("the get_byte function is called on 0x860a")  
+       THEN(""The value at the least significant byte position is "0x0a")
+       THEN("The value at the most signficant postiion is 0x86")
+
+
+   GIVEN("The util get_byte function'")
+       WHEN("the get_byte function is called on 0x86010f0a")
+       THEN("value is as expected ")
+
+
+Scenario: The Util set_byte function correctly sets bytes to the given position
+  GIVEN("The Util set_byte function ")
+  WHEN("the get_byte function is called on 0x86 for 0th index")
+     THEN("Value is as expected 134 ")
+
+  GIVEN("The Util set_byte function ")
+  WHEN("the get_byte function is called on 0x86 for 0th index")
+     THEN("Value is as expected 0 ")
+
+  GIVEN("The Util set_byte function ")
+  WHEN("the get_byte function is called on 0x83 for 0th index")
+     THEN("Value is as expected in uint16_t data-type ")
```

Comment:
Let's specify the returned values for all `THEN`s, e.g. `THEN("The returned value is 0x0083")` or `THEN("The returned value is 0x8300")`or `THEN("The returned value is 0x0000")`.